packages/react-native-renderer/src/__mocks__/react-native/Libraries/ReactPrivate/ReactNativeViewConfigRegistry.js

```diff
@@ -9,26 +9,16 @@
 
 'use strict';
 
-import type {
-  ReactNativeBaseComponentViewConfig,
-  ViewConfigGetter,
-} from './ReactNativeTypes';
+import {type ViewConfig} from './ReactNativeTypes';
 
 // Event configs
-const customBubblingEventTypes = {};
-const customDirectEventTypes = {};
-const eventTypes = {};
-
-exports.customBubblingEventTypes = customBubblingEventTypes;
-exports.customDirectEventTypes = customDirectEventTypes;
-exports.eventTypes = eventTypes;
```

Comment:
I think this also needs removed from here: https://github.com/facebook/react/blob/151e75a128d0fd436dce365335b96c5686f704d4/scripts/flow/react-native-host-hooks.js#L137

Reply:
nice find, removed that in a separate PR